components/src/preact/prevalenceOverTime/prevalence-over-time-bar-chart.tsx

```diff
@@ -20,7 +23,11 @@ const PrevalenceOverTimeBarChart = ({
     data,
     yAxisScaleType,
     confidenceIntervalMethod,
+    yAxisMaxConfig,
 }: PrevalenceOverTimeBarChartProps) => {
+    const maxY =
+        yAxisScaleType !== 'logit' ? getYAxisMax(maxInData(data), yAxisMaxConfig?.[yAxisScaleType]) : undefined;
```

Comment:
```suggestion
        yAxisScaleType !== 'logit' ? getYAxisMax(maxInData(data), yAxisMaxConfig[yAxisScaleType]) : undefined;
```
Why don't we simply require it to be present? (We could also do that in the other charts)

Reply:
The idea is to change this in #303, since it targets the logit scale. Otherwise it might me confusing to set something, which has no effect.